xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -525,36 +525,40 @@ static int Seek(const std::vector<std::string>& params)
+///     | Forward                 | Fast Forward                                | Fast Forward                |             |
+///     | Rewind                  | Rewind                                      | Rewind                      |             |
+///     | Next                    | Next chapter or movie in playlists          | Next track                  |             |
+///     | Previous                | Previous chapter or movie in playlists      | Previous track              |             |
+///     | TempoUp                 | Increases playback speed                    | none                        | Kodi v18    |
+///     | TempoDown               | Decreases playback speed                    | none                        | Kodi v18    |
+///     | BigSkipForward          | Big Skip Forward                            | Big Skip Forward            | Kodi v15    |
+///     | BigSkipBackward         | Big Skip Backward                           | Big Skip Backward           | Kodi v15    |
+///     | SmallSkipForward        | Small Skip Forward                          | Small Skip Forward          | Kodi v15    |
+///     | SmallSkipBackward       | Small Skip Backward                         | Small Skip Backward         | Kodi v15    |
+///     | SeekPercentage(n)       | Seeks to given percentage                   | Seeks to given percentage   |             |
+///     | Random *                | Toggle Random Playback                      | Toggle Random Playback      |             |
+///     | RandomOn                | Sets 'Random' to 'on'                       | Sets 'Random' to 'on'       |             |
+///     | RandomOff               | Sets 'Random' to 'off'                      | Sets 'Random' to 'off'      |             |
+///     | Repeat *                | Cycles through repeat modes                 | Cycles through repeat modes |             |
+///     | RepeatOne               | Repeats a single video                      | Repeats a single track      |             |
+///     | RepeatAll               | Repeat all videos in a list                 | Repeats all tracks in a list|             |
+///     | RepeatOff               | Sets 'Repeat' to 'off'                      | Sets 'Repeat' to 'off'      |             |
+///     | Partymode(music) **     | none                                        | Toggles music partymode     |             |
+///     | Partymode(video) **     | Toggles video partymode                     | none                        |             |
+///     | Partymode(path to .xsp) | Partymode for *.xsp-file                    | Partymode for *.xsp-file    |             |
+///     | ShowVideoMenu           | Shows the DVD/BR menu if available          | none                        |             |
+///     | FrameAdvance(n) ***     | Advancing video by a given number of frames | none                        | Kodi v18    |
```

Comment:
Don't nominalize - `Advance video [...]` is fine

Reply:
Maybe try to refer n somewhere in the description: Advance video by n frames